demo.ipynb

```diff
@@ -153,6 +154,72 @@
     "mcmc.plot_2d([['omegabh2','omegach2','H0'], ['H0','omegach2']]);"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Changing the appearance\n",
+    "\n",
+    "Anesthetic tries to follow matplotlib conventions as much as possible, so \n",
+    "most changes to the appearance should be relatively straight forward. \n",
+    "Here are some examples:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "* **figure size**:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig = plt.figure(figsize=(5, 5))\n",
+    "fig, axes = make_2d_axes(['omegabh2', 'omegach2', 'H0'], fig=fig, tex=mcmc.tex)\n",
+    "mcmc.plot_2d(axes);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "* **legends**:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig, axes = make_2d_axes(['omegabh2', 'omegach2', 'H0'], tex=mcmc.tex)\n",
+    "mcmc.plot_2d(axes, label='Posterior');\n",
+    "axes.iloc[-1, 0].legend(bbox_to_anchor=(len(axes), len(axes)), loc='upper left');"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "* **unfilled contours** &nbsp; & &nbsp; **modifying individual axes**:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig, axes = make_2d_axes(['omegabh2', 'omegach2', 'H0'], tex=mcmc.tex)\n",
+    "mcmc.plot_2d(axes.iloc[0:1, :], types=dict(upper='kde', lower='kde', diagonal='kde'), fc=None);\n",
+    "mcmc.plot_2d(axes.iloc[1:2, :], types=dict(upper='kde', lower='kde', diagonal='kde'), fc=None, cmap=plt.cm.Oranges, lw=3);\n",
+    "mcmc.plot_2d(axes.iloc[2:3, :], types=dict(upper='kde', lower='kde', diagonal='kde'), fc='C2', ec='C3', c='C4', lw=2);"
```

Comment:
This is looking a bit gaudy, hope that's ok. It does provide a quite detailed example of how to customise, though.

Reply:
This is fine, as it it quite fine-grained control, although I agree that in anesthetic 2.0 we should probably have a cleaner api. You could probably clean it up by defining `kde=dict(upper='kde', lower='kde', diagonal='kde')`, and passing `types=kde` to each of them.